spec/types/calculation.md

```diff
@@ -341,3 +347,32 @@ To evaluate a `CalcValue` production `value` into a `CalculationValue` object:
   > Allowing variables to return unquoted strings here supports referential
   > transparency, so that `$var: fn(); calc($var)` works the same as
   > `calc(fn())`.
+
+### `ParenthesizedVar`
+
+> If a `var()` is written directly within parentheses, it's necessary to
+> preserve those parentheses. CSS resolves `var()` by literally replacing the
+> function with the value of the variable and *then* parsing the surrounding
+> context.
+>
+> For example, if `--ratio: 2/3`, `calc(1 / (var(--ratio)))` is parsed as
+> `calc(1 / (2/3)) = calc(3/2)` but `calc(1 / var(--ratio))` is parsed as
+> `calc(1 / 2/3) = calc(2/3)`.
+
+To evaluate a `ParenthesizedVar` production `value` into an unquoted string:
+
+* Let `function` be a [`FunctionCall`] with `"var"` as its
+  [`NamespacedIdentifier`] and with `value`'s `ArgumentInvocation`.
+
+  [`FunctionCall`]: ../functions.md#syntax
+  [`NamespacedIdentifier`]: ../modules.md#syntax
+
+* Let `result` be the result of evaluating `function`.
+
+* If `result` is a number or a calculation, return it.
+
+  > This could happen if the user defines a `var` function in Sass.
```

Comment:
isn't that forbidden ?

Reply:
No, only names that have special parsing rules (`url`, `and`, `not`, etc) are forbidden as function names. Users are allowed to override plain CSS functions if they want.